device-schema.json

```diff
@@ -9,6 +9,30 @@
       "pattern": "^([[:xdigit:]]{2}[:.-]?){5}[[:xdigit:]]{2}$",
       "description": "Property. Model:'https://schema.org/Text'. The MAC address of the device."
     },
+    "ControlledAssetType": {
+      "type": "array",
+      "description": "Property. Model: 'https://schema.org/Text. List of asset(s) (building, object, etc.) controlled by the device",
+      "items": {
+        "oneOf": [
```

Comment:
This seems to be a relationship, this was the case then it should be coded like this:
"anyOf": [
        {
          "type": "string",
          "minLength": 1,
          "maxLength": 256,
          "pattern": "^[\\w\\-\\.\\{\\}\\$\\+\\*\\[\\]`|~^@!,:\\\\]+$",
          "description": "Property. Identifier format of any NGSI entity"
        },
        {
          "type": "string",
          "format": "uri",
          "description": "Property. Identifier format of any NGSI entity"
        }
      ],